app/qml/pages/WatchesPage.qml

```diff
@@ -84,7 +84,7 @@ Page {
         anchors.centerIn: parent   
         color: Suru.foregroundColor
         font.pointSize: units.gu(2)     
-        text: i18n.tr("No smartwatches configured yet. Please connect your smartwatch using System Settings.")
+        text: i18n.tr("No smartwatches configured yet. Please connect your smartwatch using the system settings.")
```

Comment:
Using `System Settings` here makes sense as the location to look for is in the app `System Settings`. Any reason to lowercase this here?

Reply:
If it does what it says on the tin, what relevance does it bare that it is an app (with a proper name)?
To me it sounds like some kind of TV-shop concept. "Open the SmartFridgeDoorTM on your SmartFridge2012.", only now, it is The Door door.

Reply:
Your edit actually makes your point perfectly clear :)
This makes complete sense now :+1:

Reply:
@MagneFire Well we can't have that ;)